node/src/opt.rs

```diff
@@ -186,14 +186,6 @@ pub struct Opt {
         help = "Password to use for Elasticsearch logging"
     )]
     pub elasticsearch_password: Option<String>,
-    #[clap(
-        long,
-        value_name = "MILLISECONDS",
-        default_value = "1000",
-        env = "ETHEREUM_POLLING_INTERVAL",
-        help = "How often to poll the Ethereum node for new blocks"
-    )]
-    pub ethereum_polling_interval: u64,
```

Comment:
Why did you remove this from `Opt` and put it in `EnvVars`?

Reply:
It's not bad, just curious.

Reply:
because I didn't want to add a dependency on the node package from the ethereum package and I don't think this would be used as a flag since it's eth specific.

Reply:
we could leave it in both places but if used as an argument AND var we would have to make this more complicated than it should be 😛

Reply:
Not adding the crate as a dependency is a good reason I guess, yeah I'm okay with it. Resolving.